x-pack/solutions/search/plugins/search_query_rules/public/components/query_ruleset_detail/query_rule_draggable_list/query_rule_search_results.tsx

```diff
@@ -0,0 +1,262 @@
+import { QueryRuleDraggableListItemActionTypeBadge } from './query_rule_draggable_item_action_type_badge';
+import { QueryRuleDraggableItemCriteriaDisplay } from './query_rule_draggable_item_criteria_display';
+import { DeleteRulesetRuleModal } from './delete_ruleset_rule_modal';
+import { AnalyticsEvents } from '../../../analytics/constants';
+import { DroppableContainer } from '../styles';
+
+export interface QueryRuleSearchResultsProps {
+  rules: SearchQueryRulesQueryRule[];
+  unfilteredRules: SearchQueryRulesQueryRule[];
+  queryRule: QueryRulesQueryRule;
+  rulesetId: string;
+  onEditRuleFlyoutOpen: (ruleId: string) => void;
+  deleteRule?: (ruleId: string) => void;
+  isLastItem?: boolean;
+  tourInfo?: {
+    title: string;
+    content: string;
+    tourTargetRef?: React.RefObject<HTMLDivElement>;
+  };
+}
+
+export const QueryRuleSearchResults: React.FC<QueryRuleSearchResultsProps> = ({
+  rules,
+  unfilteredRules,
+  onEditRuleFlyoutOpen,
+  deleteRule,
+  queryRule,
+}) => {
+  const { euiTheme } = useEuiTheme();
+  const useTracker = useUsageTracker();
+  const [isPopoverOpen, setIsPopoverOpen] = React.useState(false);
+  const closePopover = useCallback(() => {
+    setIsPopoverOpen(false);
+  }, []);
+  const openPopover = useCallback(() => {
+    setIsPopoverOpen(true);
+  }, []);
+  const [ruleToDelete, setRuleToDelete] = useState<string | null>(null);
+  return (
+    <>
+      {ruleToDelete && (
+        <DeleteRulesetRuleModal
+          closeDeleteModal={() => setRuleToDelete(null)}
+          onConfirm={() => {
+            if (deleteRule) {
+              deleteRule(ruleToDelete);
+            }
+          }}
+        />
+      )}
+      <EuiPanel paddingSize="s" hasShadow={false}>
+        <EuiFlexGroup alignItems="center" gutterSize="m">
+          <EuiFlexItem grow={false}>
+            {/* Custom spacer to prevent elements jumping in between search and list */}
+            <EuiPanel
+              color="transparent"
+              paddingSize="s"
+              css={css({
+                minWidth: euiTheme.base * 2,
+                minHeight: euiTheme.base * 2,
+              })}
+            />
+          </EuiFlexItem>
+          <EuiFlexItem grow={true}>
+            <EuiFlexGroup alignItems="center" responsive={false}>
+              <EuiFlexItem grow={false}>
+                <EuiNotificationBadge color="subdued">-</EuiNotificationBadge>
+              </EuiFlexItem>
+              <EuiFlexItem grow={7}>
+                <EuiFlexGroup direction="column" gutterSize="xs" responsive={false}>
+                  {Array.isArray(queryRule.criteria) ? (
+                    <>
+                      {queryRule.criteria.slice(0, 3).map((criteria, criteriaIndex) => (
+                        <EuiFlexItem key={criteria.type + criteriaIndex} grow={false}>
+                          <QueryRuleDraggableItemCriteriaDisplay criteria={criteria} />
```

Comment:
Unrelated to this change, but the query rules list component doesn't announce list items properly on voice over. It hops between "Drag Handle" announcements. Perhaps there's already an a11y issue for this